lib/matplotlib/cbook.py

```diff
@@ -361,9 +361,11 @@ class _BoundMethodProxy(object):
     Minor bugfixes by Michael Droettboom
     '''
     def __init__(self, cb):
+        self._hash = hash(cb)
+        self._callbacks = []
```

Comment:
I think this member name `_callbacks` is confusing, since the `BoundMethodProxy` itself is a callback.  If I understand correctly, it is something that is called on destruction of the `BoundMethodProxy`, so maybe call it `_destroy_callbacks`?